content/en/dashboards/widgets/timeseries.md

```diff
@@ -1,132 +1,142 @@
+Each query or formula, along with any [filtering tags][6], can be aliased. The alias overrides the display on the graph and legend, which is useful for long metric names or long lists of filters. At the end of your query or formula, click on **as...** and enter your metric alias:
 
-{{< img src="dashboards/querying/metric_alias.png" alt="metric alias" style="width:75%;" >}}
+{{< img src="dashboards/widgets/timeseries/metric_alias.png" alt="Adding an alias to a search query in the Timeseries widget editor" style="width:100%;" >}}
 
-##### Event overlay
+### Event overlay
 
-Add events from related systems to add more context to your graph. For example, you can add GitHub commits, Jenkins deploys, or Docker creation events. Expand the **Event Overlays** section and enter a query to display those events. Use the same query format as for the [Event Explorer][7], for example:
+You can add events from related systems to add more context to your graph, such as GitHub commits, Jenkins deploys, and Docker creation events. Click **Add Event Overlay** in the **Event Overlays** section and enter a query to display those events. 
+
+Use the same query format as for the [Event Explorer][7], for example:
 
 | Query                       | Description                                                |
 |-----------------------------|------------------------------------------------------------|
 | `sources:jenkins`           | Shows all events from the Jenkins source.                  |
-| `tag:role:web`              | Shows all events with the tag `role:web`.                  |
-| `tags:$<TEMPLATE_VARIABLE>` | Shows all events from the selected [Template Variable][8]. |
+| `tag:role:web`              | Shows all events with the `role:web` tag.                  |
+| `tags:$<TEMPLATE_VARIABLE>` | Shows all events from the selected [template variable][8]. |
 
 Once enabled, events are overlaid on your graphs with red bars:
 
-{{< img src="dashboards/widgets/timeseries/event_overlay.png" alt="Event overlay" style="width:75%;" >}}
+{{< img src="dashboards/widgets/timeseries/event_overlay.png" alt="Event overlay bar on a timeseries graph" style="width:100%;" >}}
 
-##### Legend configuration
+### Markers
 
-Add configurable legends on your screenboards by navigating to the legend section in the graph editor and selecting an option.
+To add markers for additional data sets, click **Add Marker** in the **Markers** section.
 
-{{< img src="dashboards/widgets/timeseries/legend-config.jpg" alt="Legend configuration" style="width:100%;" >}}
+1. Select a Line or Range and input a value or a range or values.
+2. In the **Show as** field, select an alerting status/color and choose from a solid, bold, or dashed horizontal line. 
+3. To add a label that displays on the bottom left of the timeseries widget, define a value for the Y-Axis and click the **Label** checkbox. 
 
-Options:
+### Y-Axis controls
```

Comment:
I don't know if you want to take this on now, but all of these options could use some clearer organization. If you look in the app, there are 5 options all grouped together: Event Overlays, Markers, Y-Axis Controls, Legend, and Context Links. These could go together under an H3 heading, maybe something like "Graph options."

Reply:
Also, throughout this doc, "Y-Axis" should be lowercased as "y-axis"